creator/ingest_runs/common/model.py

```diff
@@ -89,6 +94,7 @@ class Meta:
         default=State.NOT_STARTED,
         help_text="The current state of the ingest process"
     )
+    study = AbstractAttribute()
```

Comment:
Hm any reason in particular you chose to create a new attribute class with a custom getter?

The way I would usually see this done is via the property decorator on a class method:
```python
class IngestProcess(model.model):
    @property
    def study(self):
        raise NotImplementedError
```

Reply:
> Hm any reason in particular you chose to create a new attribute class with a custom getter?
> 
> The way I would usually see this done is via the property decorator on a class method:
> 
> ```python
> class IngestProcess(model.model):
>     @property
>     def study(self):
>         raise NotImplementedError
> ```

I'm not entirely sure. I was following this:
https://stackoverflow.com/a/32536493

Seems the property works the same here